democracy_club/apps/core/tests/test_html.py

```diff
@@ -65,6 +65,25 @@ def test_html_valid(self, client, subtests, urls):
                 _, errors = validate_html(client, url)
                 assert errors == ""
 
+    @pytest.fixture
+    def redirected_urls(self):
+        return [
+            "/voters/",
+            "/data/",
+            "/mailing_list/",
+            "/impact/",
+            "/data_apis/",
+            "/about/impact/",
+            "/about/support_us/",
+            "/donate/",
+        ]
+
+    @pytest.mark.django_db
+    def test_redirected_urls(self, client, redirected_urls):
+        """test redirected urls return a 301 status code"""
+        for redirect in redirected_urls:
+            assert client.get(redirect).status_code == 301 or 302
```

Comment:
This assertion asserts that either:

`client.get(redirect).status_code == 301`
or
`302`

`assert 302` will always pass because 302 is "truthy"

I think what you really want here is:

```py
"""test redirected urls return a 301 or 302 status code"""
for redirect in redirected_urls:
    status_code = client.get(redirect).status_code
    assert status_code == 301 or status_code == 302
```